### ISSUE TITLE
f.cam6_3_101.FLTHIST_v0a.ne30.001

### ISSUE THREAD

**Issue by cecilehannay** [NCAR/amwg_dev#229]
**Description:**
The purpose of this run will be to test the new compset ``FLTHIST_v0a`` (this compset is the first version of ``FLTHIST``). 

See: https://github.com/NCAR/amwg_dev/issues/226

**Case directory:**
Locally (if still available):
/glade/p/cesmdata/cseg/runs/cesm2_0/f.cam6_3_101.FLTHIST_v0a.ne30.001

On github:
https://github.com/NCAR/amwg_dev/tree/f.cam6_3_101.FLTHIST_v0a.ne30.001

**Sandbox:**
Locally (if still available):
/glade/work/hannay/cesm_tags/cam6_3_100_FLTHIST_v0a

On github:
Basically, this sandbox will be similar to cam6_3_101 if Cheryl's regression tests are successful

**Diagnostics:**
ADF diags (if available)
[https://webext.cgd.ucar.edu/FLTHIST/f.cam6_3_101.FLTHIST_v0a.ne30.001/atm/](https://webext.cgd.ucar.edu/FLTHIST/f.cam6_3_101.FLTHIST_v0a.ne30.001/atm/)

**Contacts:**
@cacraigucar, @cecilehannay, @julio, @fvitt

**Comment by cecilehannay** [comment]
We can restart this run adding CO2. Maybe we can add some of the COSP diags @brianpm wants. Also remove anything that is not needed. 

Maybe we could have a look of what is there already to make sure there is not show stopper.

**Comment by cecilehannay** [comment]
@justin-richling: could you run the ADF for  ``f.cam6_3_101.FLTHIST_v0a.ne30.001`` versus ``f.cam6_3_100.FWscHIST.ne30_L58.001`` asap. Use 5-year climos. 
 Thanks

**Comment by brianpm** [comment]
Yes, please turn on COSP and I can do some comparisons with observations. Unless we have recent runs with COSP turned on, then I think we need to compare errors with respect to observations relative to CESM2-CAM6 used for CMIP runs and/or the PPE. If there's a need to reduce the output data volume, I can send a list of variables.

**Comment by cacraigucar** [comment]
@cecilehannay - is it possible for you to use cam6_3_101 when you start up these next runs?

**Comment by justin-richling** [comment]
> @justin-richling: could you run the ADF for `f.cam6_3_101.FLTHIST_v0a.ne30.001` versus `f.cam6_3_100.FWscHIST.ne30_L58.001` asap. Use 5-year climos. Thanks

Will do

**Comment by adamrher** [comment]
Interesting differences at the top of the model. [It's over 10K hotter up there](https://webext.cgd.ucar.edu/FLTHIST/f.cam6_3_101.FLTHIST_v0a.ne30.001/atm/f.cam6_3_101.FLTHIST_v0a.ne30.001_1979_1985_vs_f.cam6_3_100.FWscHIST.ne30_L58.001_1979_1985/html_img/plot_page_T_ANN_Zonal_Mean.html). This appears to [lower the zonal wind speeds](https://webext.cgd.ucar.edu/FLTHIST/f.cam6_3_101.FLTHIST_v0a.ne30.001/atm/f.cam6_3_101.FLTHIST_v0a.ne30.001_1979_1985_vs_f.cam6_3_100.FWscHIST.ne30_L58.001_1979_1985/html_img/plot_page_U_ANN_Zonal_Mean.html), possibly through changes to the thermal wind balance.

My number one suspect is the H2O UBC boundary conditions. Here's the fractional change in Q:

![temp_dzonal_Q](https://user-images.githubusercontent.com/29961476/229259704-90b4467a-64cf-4dc4-89da-8d80432ea79d.png)

This is consistent with Dan's original analysis of the UBC's; it hydrates the model top, but this is attenuated away with greater distance from the model top. There's even a reversal suggesting a significantly drier UTLS (note: yellow is the zero contour). This may not be due to the UBC's alone ... it could result from other changes in ghg_mam4.

I would guess that what we are seeing in FLTHIST is greater SW radiative heating due to the additional water vapor at the model top. Not seeing much evidence for a LW effect (e.g., see FLNT in Justin's diagnostics). It would be good to verify the other GHG's up there, but we'll have to wait until the next run w/ CO2 output.

@islasimpson @dan800

**Comment by dan800** [comment]
Thanks, Adam. Do we have QRS and QRL in the output?DanSent from my iPhoneOn Apr 1, 2023, at 3:53 AM, Adam Herrington ***@***.***> wrote:﻿
Interesting differences at the top of the model. It's over 10K hotter up there. This appears to lower the zonal wind speeds, possibly through changes to the thermal wind balance.
My number one suspect is the H2O UBC boundary conditions. Here's the fractional change in Q:

This is consistent with Dan's original analysis of the UBC's; it hydrates the model top, but this is attenuated away with greater distance from the model top. There's even a reversal suggesting a significantly drier UTLS (note: yellow is the zero contour). This may not be due to the UBC's alone ... it could result from other changes in ghg_mam4.
I would guess that what we are seeing in FLTHIST is greater SW radiative heating due to the additional water vapor at the model top. Not seeing much evidence for a LW effect (e.g., see FLNT in Justin's diagnostics). It would be good to verify the other GHG's up there, but we'll have to wait until the next run w/ CO2 output.
@islasimpson @dan800

**Comment by cecilehannay** [comment]
About COSP simulator.

I tried to run with the COSP simulator:
- adding ``-cosp`` in CAM_CONFIG_OPTS

but I got the error:
```
(GETFIL): attempting to find local file iceoptics_c080917.nc
 (GETFIL): using
 /glade/p/cesmdata/cseg/inputdata/atm/cam/physprops/iceoptics_c080917.nc
  reading ice cloud optics from file
 /glade/p/cesmdata/cseg/inputdata/atm/cam/physprops/iceoptics_c080917.nc        
                                                                               
                                                                               
                   
 (GETFIL): attempting to find local file water_refindex_rrtmg_c080910.nc
 (GETFIL): using
 /glade/p/cesmdata/cseg/inputdata/atm/cam/physprops/water_refindex_rrtmg_c080910
 .nc
 modal_aer_opt_init: read water refractive index file:
 /glade/p/cesmdata/cseg/inputdata/atm/cam/physprops/water_refindex_rrtmg_c080910
 .nc
 ERROR: Attempt to find undefined name in pbuf SH_CLDLIQ
```

**Comment by adamrher** [comment]
I believe @bstephens82 has the bug fix for this issue in his PR https://github.com/ESCOMP/CAM/pull/777

**Comment by adamrher** [comment]
It does look like the heating of the model top is due to QRL; it increases to about a +0.1 K/day.

![temp_dzonal_QRL](https://user-images.githubusercontent.com/29961476/229298145-1146937a-cd33-4bf5-a00a-f7d03dc08271.png)

![temp_dzonal_PTTEND](https://user-images.githubusercontent.com/29961476/229298149-de369c94-15e0-490a-ba12-cb3e47a3859e.png)

![temp_dzonal_T](https://user-images.githubusercontent.com/29961476/229298155-6b6c639c-abe5-4dd3-be9a-540e98184934.png)

**Comment by cecilehannay** [comment]
> I believe @bstephens82 has the bug fix for this issue in his PR [ESCOMP/CAM#777](https://github.com/ESCOMP/CAM/pull/777)
> 
> [Ben can you package up some SourceMods for Cecile? This is for cam tag 101.]

I implemented the changes from that PR [ESCOMP/CAM#777] 
https://github.com/ESCOMP/CAM/pull/777/files/ea7f5af369ac175499dfb3894626481f662366df#
into the SourceMods from:
``/glade/p/cesmdata/cseg/runs/cesm2_0/f.cam6_3_101.FLTHIST_v0a.ne30.002``

It seems to be running.

**Comment by dan800** [comment]
> It does look like the heating of the model top is due to QRL; it increases to about a +0.1 K/day.
> 
> 
> 
> ![temp_dzonal_QRL](https://user-images.githubusercontent.com/29961476/229298145-1146937a-cd33-4bf5-a00a-f7d03dc08271.png)
> 
> 
> 
> ![temp_dzonal_PTTEND](https://user-images.githubusercontent.com/29961476/229298149-de369c94-15e0-490a-ba12-cb3e47a3859e.png)
> 
> 
> 
> ![temp_dzonal_T](https://user-images.githubusercontent.com/29961476/229298155-6b6c639c-abe5-4dd3-be9a-540e98184934.png)

If it is warmer QRL goes up, so necessarily the cause. QRS is where to look first IMO.

**Comment by adamrher** [comment]
> If it is warmer QRL goes up, so necessarily the cause. QRS is where to look first IMO.

That is the first thing I looked at; QRS is not it. I'm confident it is QRL, since the QRL changes largely resemble the PTTEND changes. The only other argument left is that it's the dynamics, since DTCORE and PTTEND need to roughly balance. But I think Occam's razor supports this being a radiative response to the additional water vapor up there.

**Comment by dan800** [comment]
@adamrher A few things I don't understand with water vapor being the cause: 1) a 10K diff at about 0.1 K/day would be a radiative lifetime of 100 days, right? Isn't that 10X reality? 2) An increase in H2O should cool the stratosphere, not warm it. 

Are we sure the choice of heating from gravity wave momentum deposition is the same in these two runs?

**Comment by klindsay28** [comment]
It looks like `CO2` was initialized to 0 and the non-zero prescribed concentration surface boundary condition propagated non-zero values up as the run progressed. I wouldn't expect `CO2` to be initialized this way. Is it possible to initialize this `CO2` constituent to a non-zero value?

**Comment by klindsay28** [comment]
Whoops. Wrong GH issue. I was looking at values in f.cam6_3_101.FLTHIST_v0a.ne30.002.

**Comment by adamrher** [comment]
Ah OK. I'll move over there with ya. @dan800 this may complicate our interpretation.
